README.md

```diff
@@ -0,0 +1,111 @@
+# Stack-packet
+
+## Overview
+
+This `stack-packet` repository is the implementation of a Crossplane infrastructure for
+[Packet](https://www.packet.com).
+The stack that is built from the source code in this repository can be installed into a Crossplane control plane.
+
+## Getting Started and Documentation
+
+For getting started guides, installation, deployment, and administration, see [Documentation](https://crossplane.io/docs/latest).
+
+## Pre-requisites 
+
+* Kubernetes cluster
+    * For example Minikube, minimum version v0.28+
+* Helm, minimum version v2.9.1+.
+
+## Installing Crossplane
+
+Add Helm chart repo:
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+```
+Install Crossplane Helm chart
+
+```
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+For more details on Crossplane installation visit [here](https://crossplane.io/docs/v0.1/install-crossplane.html)
+
+## Installing `stack-packet` 
+
+* Create CRDs for the stack:
```

Comment:
With the [Crossplane CLI installed](https://github.com/crossplaneio/crossplane-cli#installation), the non-developer approach to installing the Stack would be:
 
```
kubectl crossplane stack install --cluster packethost/stack-packet
```

A requirement for offering this approach would be the existence of a stack-packet image at https://hub.docker.com/u/packethost

Reply:
@displague we are working on it.